joomla/joomla-cms
# [3.5] [Imp] Remove useless tags sidebar

### What is changed?

This PR removes the useless Sidebar for Tags and fixes some mirror CS
### Bevor the change

![bevor_tags_removal](https://cloud.githubusercontent.com/assets/2596554/9046742/2b1b6aec-3a2d-11e5-808f-08877726bfe3.PNG)
### After the change

![after_tags_removal](https://cloud.githubusercontent.com/assets/2596554/9046746/30937bf4-3a2d-11e5-84a9-eba7b908e55a.PNG)
### How to test
- install 3.5
- access com tags
- see `Bevor the change`
- apply the patch
- access com tags
- see `After the change`
